wremnants/datasets/datagroups.py

```diff
@@ -279,7 +279,7 @@ def __init__(self, infile, combine=False, pseudodata_pdfset = None,
     ):
         self.datasets = {x.name : x for x in datasets2016.getDatasets()}
         super().__init__(infile, combine)
-        self.wlike = "wlike" in self.results["meta_info"]["command"]
+        self.wlike = "wlike" in self.results["meta_info"]["command"] or "dilepton" in self.results["meta_info"]["command"]
```

Comment:
We should make this check a bit more robust. Maybe something like ```"mz" in os.path.basename(self.results["meta_info"]["command"]).split()[:2]``` ...? It's janky, but in the current setup, if someone happened to use "wlike" or "dilepton" in the name of the output it would trigger this

Reply:
I agree. I'm further thinking to promote the dilepton equal to wlike, right now wlike is often used for dilepton, which is confusing some times and needs hacks at some points. We could add self.dilepton at this place and other modification elsewhere